lib/Backends/CPU/libjit/libjit.cpp

```diff
@@ -725,6 +727,39 @@ libjit_space_to_depth_generic(const T *inPtr, T *outPtr, size_t blockSize,
     }
   }
 }
+/// The dimensions passed in here are pre-expanded in LLVMIRGen with 1s so that
+/// we can iterate over the shape here, regardless of the shape of the tensor.
+template <typename T>
+static void libjit_reducemin(T *dest, const T *batch, size_t destSize,
```

Comment:
reducemin vs reducesum should be pretty much the same.
Can you unify implementations with the reduction and initialization functions to apply as params?

Reply:
Do you mean libjit_batchedreduceadd_f? I probably won't have bandwidth to do that and add tests as well for new data-types. I'd prefer to keep this ReduceMin only change and rather not touch other paths.